src/Parser/Monk/Brewmaster/CHANGELOG.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
-import { WOPR, Zerotorescue } from 'MAINTAINERS';
+import { WOPR, Zerotorescue, emallson } from 'MAINTAINERS';
 
 export default [
   {
```

Comment:
If you wish to give maintaining the spec a go feel free to remove this message (perhaps announce you'll be working on it) and update the maintainer and status in the config.